.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - v*
+  workflow_dispatch:
+
+jobs:
+  release:
+    if: github.repository == '84codes/rubocop-eighty-four-codes'
```

Comment:
is this if we would fork?

Reply:
This is fork protection, yes

Reply:
It is just a nice thing to anyone that would fork